HW10/read.txt

```diff
@@ -0,0 +1,101 @@
+var b = [5, 10, 23, 10, 7, 9, 39, 16, 1, 12, 36, 20, 50];
```

Comment:
you are looping through the array for each condition, I would start one loop and combine all if conditions inside it.